### ISSUE TITLE
GSS - local variables in mixin

### ISSUE THREAD

**Issue by slavap** [google/closure-stylesheets#17]
It should be possible to declare and use local variable somehow in mixin.
Right now it's possible declare @def in mixin, BUT it will be global variable. 
As workaround I can prefix it with mixin name to prevent names conflict, i.e. @def MIXIN_LOCAL 33; but it's very inconvenient.

**Comment by iflan** [comment]
Yeah, this is not awesome. There are several possible semantics for a `@def` in a `@mixin`, so if you have a proposition and are interested in implementing it, let me know.
